bedrock/releasenotes/views.py

```diff
@@ -211,6 +211,9 @@ def releases_index(request, product):
     if product == 'Firefox':
         major_releases = firefox_desktop.firefox_history_major_releases
         minor_releases = firefox_desktop.firefox_history_stability_releases
+        # Firefox 50.1.0 is not an ESR version but should be fallen under 50.0,
+        # so put it in the list as an exception (Bug 1324305).
+        esr_major_versions.append(50)
```

Comment:
Agree it would be preferable to provide a fix for potential future point releases as well (e.g. 54.1.0).